### ISSUE TITLE
Unable to read socket.ini: Error: ENOENT: no such file or directory, open 'socket.ini'

### ISSUE THREAD

**Issue by gniezen** [socketsupply/socket#192]
### What OS are you using (`uname -a`, or Windows version)?

`Linux 5.15.0-60-generic #66~20.04.1-Ubuntu SMP Wed Jan 25 09:41:30 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux`

### What programming language are you using (C/C++/Go/Rust)?

- `gcc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0`
- `clang version 10.0.0-4ubuntu1`
- `go version go1.13.8 linux/amd64`
- `rustc 1.66.0-nightly (f5193a9fc 2022-09-25)`

### What did you expect to see and what you saw instead?

I was following the Getting Started guide, by first installing the Socket Runtime using `npm install @socketsupply/socket -g` and then running `npx create-socket-app`. I was expecting to see the results in the Getting Started guide, but got:

```
Initializing npm package...OK
Installing developer dependencies...OK
Installing dependencies...OK
Adding package scripts...OK
Creating socket files...OK
Updating project configuration...
Unable to read socket.ini: Error: ENOENT: no such file or directory, open 'socket.ini'
```

**Comment by lamiazar** [comment]
@gniezen you can take a look at this issue: 
https://github.com/socketsupply/create-socket-app/issues/17 , and follow the steps, I think I had the same issue recently,now resolved.

**Comment by gniezen** [comment]
Thanks @lamiazar ! I'm still on Ubuntu 20.04.1 (focal) which does not have the `libwebkit2gtk-4.1-dev` package. I tried `libwebkit2gtk-4.0-dev`, which is available on Ubuntu 20.04, but it still resulted in the same error. I guess I'm out of luck until I upgrade my OS.

**Comment by chicoxyzzy** [comment]
I'm going to close this issue. We've released a beta, that should fix the issue. Feel free to reopen it if the problem will appear again.
